plugins/cri/images/plugin_test.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 func TestSandboxImageConfigMigration(t *testing.T) {
-	image := "rancher/mirrored-pause:3.9-amd64"
+	image := "rancher/mirrored-pause:3.10-amd64"
```

Comment:
Is the image updated to this registry yet?

Reply:
oh, not yet. I've changed it back, thank you.

Reply:
OK, I have changed it to 3.10, thanks

Reply:
https://github.com/rancher/image-mirror/pull/635 to make it available in the registry